mysql/datadog_checks/mysql/mysql.py

```diff
@@ -147,6 +150,15 @@ def check_performance_schema_enabled(self, db):
 
         return self.performance_schema_enabled
 
+    def check_userstat_enabled(self, db):
+        if self.userstat_enabled is None:
+            with closing(db.cursor()) as cursor:
+                cursor.execute("SHOW VARIABLES LIKE 'userstat'")
```

Comment:
I tried setting this on a regular mysql instance (mysql 8), and got an error:
```SET GLOBAL userstat=1;
ERROR 1193 (HY000): Unknown system variable 'userstat'```

Is this a variable only available in percona/ mariadb? If so that should be documented